app/operations/story_operations/update_all.rb

```diff
@@ -0,0 +1,30 @@
+module StoryOperations
+  class UpdateAll
+    include Dry::Monads[:result, :do]
+
+    def call(stories:, data:, current_user:)
+      stories = yield update_stories(
+        stories: stories,
+        data: data,
+        current_user: current_user
+      )
+
+      Success(stories)
+    rescue
+      Failure(false)
+    end
+
+    private
+
+    # TODO: we should probably use a transaction here
```

Comment:
so why not wrap this block within a transaction block?

Reply:
idk :smile: haha

Reply:
@tonyaraujop the original implementation was not using a transaction and we're avoiding refactoring two different things at the same time. So we left this comment to remember it later.